src/main.py

```diff
@@ -1,9 +1,21 @@
+import subprocess
+
 from follower import Follower
 from leader import Leader
 from tagrec import TagRecognition
 
+'''
+Calls hcitool and reads the exit value returned. hcitool returns 0 on success 
+and 1 on failure.
+
+Input: Bluetooth address.
+Output: Boolean values, true if connected false if disconnected.
+'''
+def is_connected(bt_addr):
+    return not subprocess.call(['hcitool','info', bt_addr])
+
 
-"""
+'''
```

Comment:
Don't change these from double to single quotes. It doesn't matter which we use really, but we should be consistent in a file. That means you use the ones that are already there. Don't change the rest of them,